src/pheval/cli_pheval.py

```diff
@@ -35,13 +38,15 @@
     metavar="TMPDIR",
     required=False,
     help="The path of the temporary directory (optional)",
+    type=Path,
 )
 @click.option(
     "--output-dir",
     "-o",
     metavar="OUTPUTDIR",
     required=True,
     help="The path of the output directory",
+    type=Path,
 )
 @click.option(
     "--config",
```

Comment:
I know this is not what the PR is about, but have you thought about my suggestion to ditch this?

Reply:
Ditching the ```--config``` option? I think I thought about it too much and confused myself with what you meant. Do you mean that the config attached with the ```--input-dir``` parameter would have the information contained within the config attached to the ```--config-file``` ? 

I don't know if it could be 'fixed' in terms of it's contents as the different implementations require different levels of information to run, for example, LIRICAL needs both a path to its own database and the path to the Exomiser variants database for it to output variant prioritisation results (but I guess this information would be attached to the ```--input-dir``` config anyway?). If the MakeFile is dealing with the retrieval of the databases for different implementations and can write these paths to the ```--input-dir``` config automatically then I think it's doable

Reply:
Lets talk about this tomorrow, can you add to agenda? Nothing to do with this PR here. No need to wait

Reply:
No problem!